safe/sync.go

```diff
@@ -0,0 +1,35 @@
+package safe
+
+import (
+	"fmt"
+)
+
+func (s *Safe) SyncWith(otherSafe *Safe) error {
```

Comment:
`SyncWith` is a little ambiguous to me in this instance. I think a more explicit name like `ImportFrom` or `ImportSafe` would help, or at the very least a comment clarifying the behavior of `Sync` in this context.